.github/workflows/clean_ami.yml

```diff
@@ -1,18 +1,19 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: MIT
 
-name: Monthly Resources Cleaner
+name: Clean AMI Monthly Resources Cleaner
```

Comment:
Nitpick: I think the name is redundant. Personally, I think we can drop the "Clean" at the beginning of the name